config/ecs/container-insights/otel-agent-default-config.yaml

```diff
@@ -0,0 +1,46 @@
+extensions:
+  health_check:
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: localhost:55680
+      http:
+        endpoint: localhost:55681
+  awsxray:
+    endpoint: 0.0.0.0:2000
+    transport: udp
```

Comment:
Same here. Do we want to add the default config. I think its good to avoid adding the fields which are default. It will keep the file short.